index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Maisiayonak Vitali</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+  <header>
+    <div class="container head">
+      <img src="avatar.jpg" alt="Vitali Maisiayonak">
+      <h1>Maisiayonak Vitali</h1>
+
+      <ul>
+        <li>Email:
+          <a href="mailto:moiseenokv@gmail.com">moiseenokv@gmail.com</a>
+        </li>
+        <li>Phone:
+          <a href="tel:375293224002">+375-29-322-40-02</a>
+        </li>
+        <li>Viber:
+          <a href="viber://chat?number=375293224002">+375-29-322-40-02</a>
+        </li>
+      </ul>
+    </div>
+  </header>
+  <main>
+    <div class="container">
+      <!-- Summary block begin-->
+      <div class="block summary">
```

Comment:
looks more like section or article

Reply:
OK! Thank you! I`ll keep in mind next time